integration/js/pages/AppPage.js

```diff
@@ -328,6 +329,21 @@ class AppPage {
     return 'failed'
   }
 
+  async waitForPageToLoad() {
+    let timeout = 10;
+    let i = 0;
+    let loading = false;
```

Comment:
The variable should be name something like: `finishedLoading`. `loading` indicates a running process as false from the start which is the opposite of what the code is doing.

Reply:
Will change